python/az/aro/azext_aro/_validators.py

```diff
@@ -40,15 +40,6 @@ def validate_client_id(namespace):
         except ValueError:
             raise InvalidArgumentValueError("Invalid --client-id '%s'." % namespace.client_id)
 
-        if namespace.client_secret is None or not str(namespace.client_secret):
```

Comment:
this change adversely affects validation in aro_create.

Reply:
Is this not expected? I dont get what you want to say here

Reply:
I think im rolling this back as per "being pragmatic comment"